force-app/main/default/permissionsets/Book_Access.permissionset-meta.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PermissionSet xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Specifies who has access to the Cheltham library books</description>
+    <hasActivationRequired>false</hasActivationRequired>
+    <label>Book Access</label>
```

Comment:
Is this referring to permission set access to the Book object?

As we add new objects to the project they will be adding their permission set settings to this permission set. A better name could be "Admin Permission Set" or "Admin Object Access".

Reply:
Permission has been renamed to 'Admin Access'